pkg/checker.go

```diff
@@ -0,0 +1,170 @@
+package pkg
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"golang.org/x/net/html"
+)
+
+const (
+	SuccessColor = "\033[1;32m%s\033[0m"
+	ErrorColor   = "\033[1;31m%s\033[0m"
+	DebugColor   = "\033[0;36m%s\033[0m"
+)
+
+/*type URLStatus struct {
+	url    string
+	err    string
+	status int
+	index  int
+}*/
+
+type URLStatus struct {
+
+	// err
+	Err string `json:"err,omitempty"`
+
+	// index
+	Index int64 `json:"index,omitempty"`
+
+	// status
+	Status int64 `json:"status,omitempty"`
+
+	// url
+	URL string `json:"url,omitempty"`
+}
+
+func checkWebsite(website string) (string, error) {
```

Comment:
please make sure the schema `https://` is prefixing the url, if not, ensure it, and if the link starts with `http://` report it as broken link. All websites should be referencing `https` links. No need to check 3 versions of the website, 

also there's no protection against

```https://https://google.com``` for instance